### PR TITLE
Feat/add forgot password

### PR DESCRIPTION
**Pull request by Alb93**

### CODE REVIEW

File: packages/game-app/src/forgotPassword/components/ForgotPassword/ForgotPassword.tsx [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+import { Box, Button, ErrorMessage, Link, TowColumnPage, Typography } from '@pipeline/components';
+import { useTranslate } from '@pipeline/i18n';
+import { useSendResetPasswordEmail } from '@pipeline/auth';
+import { useLocation } from 'react-router-dom';
+import { RoutingPath, useNavigateTo } from '@pipeline/routing';
+import styled from 'styled-components';
+
+type Props = {};
+
+const GoToSpan = styled.span`
```
Comment (rams23) [inline packages/game-app/src/forgotPassword/components/ForgotPassword/ForgotPassword.tsx:13]: typography? chek variants

File: packages/game-app/src/forgotPassword/components/ForgotPassword/ForgotPassword.tsx [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+              </form>
+            </FormProvider>
+          </Box>
+        </>
+      }
+    />
+  );
+};
+
+ForgotPassword.displayName = 'Login';
```
Comment (rams23) [inline packages/game-app/src/forgotPassword/components/ForgotPassword/ForgotPassword.tsx:83]: change display name

File: packages/game-app/src/forgotPassword/components/ResetPassword/ResetPassword.tsx [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+import { LinkUrlParams } from '@pipeline/models';
+import { useResetPassword, useVerifyActionCode } from '@pipeline/auth';
+import styled from 'styled-components';
+
+type Props = {};
+
+const GoToSpan = styled.span`
+  font-size: 16px;
+`;
+GoToSpan.displayName = 'GoToSpan';
```
Comment (rams23) [inline packages/game-app/src/forgotPassword/components/ResetPassword/ResetPassword.tsx:19]: up

File: packages/game-app/src/forgotPassword/components/ResetPassword/ResetPassword.tsx [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+              </form>
+            </FormProvider>
+          </Box>
+        </>
+      }
+    />
+  );
+};
+
+ResetPassword.displayName = 'Login';
```
Comment (rams23) [inline packages/game-app/src/forgotPassword/components/ResetPassword/ResetPassword.tsx:107]: up

**Comment by Alb93** [comment]
TODO cypress tests for forgot password

**Pull request merged**